RELEASE_NOTES.md

```diff
@@ -24,6 +24,81 @@ Chef now ships with Ruby 2.6.2. This new version of Ruby improves performance an
 
 ## Breaking Changes
 
+### Knife Bootstrap
```

Comment:
This is more of a todo comment, but before releasing this we should work on rewording this release note item to focus on the positive changes. This is a huge win for users that also comes with some breaking changes. We should be super proud of all the cool parts of that work, while also making sure to warn people that there's some breaking bits.